src/components/App.jsx

```diff
@@ -23,6 +23,10 @@ const ReactHint = ReactHintFactory(React)
 const App = props => {
   const settings = getDefaultedSettingsFromCollection(props.settingsCollection)
   flag('local-model-override', settings.community.localModelOverride.enabled)
+  flag(
+    'late-health-reimbursement-limit',
+    settings.notifications.lateHealthReimbursement.value
```

Comment:
Should we have  `|| 30` here ?

Reply:
The settings collection has default values thanks to the line 24:

```js
const settings = getDefaultedSettingsFromCollection(props.settingsCollection)
```

In these default values, we have `30`.